docs/manual.rst

```diff
@@ -1699,22 +1699,25 @@ linestring feature (right).
   Returns a LineString or MultiLineString geometry at a distance from the
   object on its right or its left side.
 
-  Alias method to :meth:`offset_curve` method.
+  Alias method to :meth:`offset_curve` method, but uses `resolution` instead
```

Comment:
Aliases with different parameters feels like a weird pattern.  Instead, it seems better to have this as a pure alias, and then having `resolution` and `'left'` / `'right'` values as deprecated for shapely 2.0.

Reply:
But shouldn't we then just deprecate the whole method? Because if we ask users to no longer pass the `side` keyword, they need to update their code anyway, and they can just as well change the method name?

Reply:
Deprecating the method seems reasonable.  To be clear: I don't have strong opinions about this, only that aliases that are not identical to the thing they are aliasing are not really aliases...

Reply:
>  only that aliases that are not identical to the thing they are aliasing are not really aliases...

I can also not call it an "alias" ;)